rollup/default.js

```diff
@@ -0,0 +1,91 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import path from 'path';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+
+export default [
+	{
+		input: 'src/index.js',
+		output: [
+			{ file: 'dist/index.cjs', format: 'cjs', sourcemap: true, exports: 'auto' },
+			{ file: 'dist/index.mjs', format: 'esm', sourcemap: true, exports: 'auto' },
+		],
+		onwarn: (warning) => {
+			// Silence circular dependency warning for postcss-values-parsers package
+			if (
+				warning.code === 'CIRCULAR_DEPENDENCY' &&
+				warning.importer.indexOf('node_modules/postcss-values-parser/lib') > -1
+			) {
+				return;
+			}
+
+			console.warn(`(!) ${warning.message}`);
+		},
+		plugins: [
+			babel({
+				babelHelpers: 'bundled',
+				exclude: 'node_modules/**',
+				presets: [
+					['@babel/preset-env', {
+						corejs: 3,
+						loose: true,
+						modules: false,
+						targets: { node: 12 },
+						useBuiltIns: 'usage',
+					}],
+				],
+			}),
+			terser(),
+		],
+	},
+	{
+		input: 'src/cli.js',
+		output: [
+			{ file: 'dist/cli.mjs', format: 'esm', sourcemap: false },
+		],
+		onwarn: (warning) => {
+			// Silence circular dependency warning for postcss-values-parsers package
```

Comment:
Can you outline where the circular dependency is? I'd love to get rid of this workaround if doable

Reply:
I opened an issue to outline this later once the general setup is complete : https://github.com/csstools/postcss-plugins/issues/8

At the moment I can't run it to trigger the warning again.

Reply:
Thanks for doing that!